include/swift/Option/FrontendOptions.td

```diff
@@ -432,6 +432,10 @@ def solver_enable_operator_designated_types :
   Flag<["-"], "solver-enable-operator-designated-types">,
   HelpText<"Enable operator designated types in constraint solver">;
 
+def diagnose_invalid_ephemeralness_as_error :
+  Flag<["-"], "diagnose-invalid-ephemeralness-as-error">,
+  HelpText<"Diagnose invalid ephemeral to non-ephemeral conversions as errors">;
```

Comment:
It might be nice™ to phrase this as `-enable…` so that you can add a `-disable…` as well.

Reply:
@jrose-apple Hmm, I only really intended for this to be a temporary flag for testing the error behaviour, and was planning on removing it once we error-by-default. Do you think it would be worth keeping an option to downgrade to a warning?

Reply:
It might help with the initial adoption within Apple, even if we eventually take the flag away.